azurerm/resource_arm_application_gateway.go

```diff
@@ -50,6 +50,40 @@ func resourceArmApplicationGateway() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"identity": {
```

Comment:
could we [add this new block to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/application_gateway.html.markdown#L171)?

Reply:
OK